Formula/fastfetch.rb

```diff
@@ -25,10 +25,29 @@ class Fastfetch < Formula
 
   uses_from_macos "zlib" => :build
 
+  on_macos do
+    depends_on "molten-vk" => :build
+  end
+
+  on_linux do
+    depends_on "dbus" => :build
+    depends_on "json-c" => :build
+    depends_on "libx11" => :build
+    depends_on "libxcb" => :build
+    depends_on "libxrandr" => :build
+    depends_on "mesa" => :build
+    depends_on "opencl-icd-loader" => :build
+    depends_on "pulseaudio" => :build
+    depends_on "rpm" => :build
+    depends_on "sqlite" => :build
+    depends_on "wayland" => :build
+  end
```

Comment:
Could you explain why we need all these?

Reply:
According to the [its README](https://github.com/LinusDierheimer/fastfetch/blob/52eacd71407ccf397c1ae427d3db5d5fc8a902f0/README.md?plain=1#L130), if the headers for dependencies are missing (i.e. not found using `pkg-config`), it will build without support for that feature. This also means that runtime library lookups are disabled.

I removed the `molten-vk` in `on_macos` block as it seems to be only needed when using Vulkan features at runtime, regardless of build.